app/retail/views.py

```diff
@@ -496,6 +504,70 @@ def browser_extension_firefox(request):
     return redirect('https://addons.mozilla.org/en-US/firefox/addon/gitcoin/')
 
 
+def idea_show(request, idea_id):
+    return TemplateResponse(request, 'idea.html', {})
+
+
+@csrf_exempt
+def new_idea(request):
+    return TemplateResponse(request, 'new_idea.html', {})
```

Comment:
We should use the same view for `new_idea` and `create_idea`, handling create on POST, and render page on GET.

Reply:
ok, i get it!